src/algebra/group_power/order.lean

```diff
@@ -371,6 +377,14 @@ lemma pow_lt_self_of_lt_one (h₀ : 0 < a) (h₁ : a < 1) (hn : 1 < n) : a ^ n <
 calc a ^ n < a ^ 1 : pow_lt_pow_of_lt_one h₀ h₁ hn
 ... = a : pow_one _
 
+lemma pow_sub_one_eq_mul_pow_sub_two (n : R) {k : ℕ} (hk : 1 < k) : n^(k - 1) = n * n^(k - 2) :=
+begin
+  conv_rhs { conv { congr, rw ←pow_one n, }, rw [←pow_add, add_comm], },
+  rw [nat.sub_succ k 1, ←nat.succ_eq_add_one,
+    nat.succ_pred_eq_of_pos (lt_tsub_iff_right.2 _)],
+  apply hk,
+end
+
```

Comment:
This lemma feels pretty contrived to me; I would expect the caller to rewrite `(k - 1)` as `(k - 2).succ` and then use `pow_succ`.